functions/postSurveyInit.ts

```diff
@@ -152,7 +190,17 @@ export const handler = TokenValidator(
       if (channelSid === undefined) return resolve(error400('channelSid'));
       if (taskSid === undefined) return resolve(error400('taskSid'));
 
-      await postSurveyInitHandler(context, { channelSid, taskSid, taskLanguage });
+      const result = await postSurveyInitHandler(context, {
+        channelSid,
+        taskSid,
+        taskLanguage: taskLanguage || 'en-US',
+      });
```

Comment:
Just a question: Would it make sense to move the fallback `en-US` to Studio? I think this fallback wouldn't be very useful for helplines that don't operate in English, for example.

Reply:
You are right. In that case we should "fail" if language is not provided, which is not a bad thing at all.